test/converters/converters.go

```diff
@@ -0,0 +1,21 @@
+package converters
```

Comment:
Package name is a bit too generic IMO.

Reply:
yeah, tried to be as generic as possible

Reply:
This is the most common name I could find for the collection of converters.